cli/spec/support/output_helpers.rb

```diff
@@ -52,23 +54,25 @@ module OutputHelpers
         line = 0
         @real.zip(@expected) do |real, expected|
           line += 1
-          error = values_match?(real.split(/\S+/), expected)
-          if error
-            @error ||= ""
-            @error << "\n#{error}"
-            @error.strip!
+          fields = real.split(/\s{2,}/)
```

Comment:
For now it seems to work (a way better than the previous one, I wonder how it worked at all) as long as no field has two spaces in it.